src/Selenium2Driver.php

```diff
@@ -131,7 +131,7 @@ public function setDesiredCapabilities($desiredCapabilities = null)
         // See https://sites.google.com/a/chromium.org/chromedriver/capabilities
         if (isset($desiredCapabilities['chrome'])) {
 
-            $chromeOptions = array();
+            $chromeOptions = (isset($desiredCapabilities['goog:chromeOptions']) && is_array($desiredCapabilities['goog:chromeOptions']))? $desiredCapabilities['goog:chromeOptions']:array();
```

Comment:
I'm not sure if we should rename keys that come from `.behat.yml`. Maybe `:` in YAML keys isn't even supported.

Reply:
Yes, this change introduce a backward compatibility break. It is possible to keep the older option name and change the option name here only after the process.
What doing ?

Reply:
Since before your PR options weren't passed at all, then no BC break really :) Looks good.